main.py

```diff
@@ -35,11 +35,11 @@ def reEmail(fname):
 #You should use a ( and ) regex like " (extract this) " to extract a portion of the match
 
 def reAward(fname):
-  fhand = open(fname, 'r')
+  fhand = open(fname,'r')
   for line in fhand:
-    extr = re.findall('.*', line)
-    if len(extr):
-      print(extr)
+      extr = re.findall('-(.*)',line)
```

Comment:
You extract an extra space before the award winner name. 

`extr = re.findall('- (.*)', line)`